lib/parser.js

```diff
@@ -23,7 +23,7 @@ function merge(query, next) {
 }
 
 function parseFilter (filter) {
-	let filterConf = /^([^\!\~\?]*)(\!\?|\!=|=|>|<|\~|\?)([^=\~\?]*)$/.exec(filter);
+	let filterConf = /^([^\!\~\?]*)(\!\?|\!=|=|>|<|\~|\!\~|\?)([^=\~\?]*)$/.exec(filter);
```

Comment:
Ah no, am being overly cautious http://www.regexr.com/3csn4

In which case I suppose I prefer the way you did it, but would prefer if you've move `!?` to be next to `?` in that case, for consistency